middleware.js

```diff
@@ -298,6 +298,21 @@ async function requireNoOptOut(req, res, next) {
   }
 }
 
+async function requireMozAdmin(req, res, next) {
+  const user = req.user;
+
+  if (!user?.primary_email.includes("@mozilla.com")) {
+    console.error("non mozilla email");
+    return res.status(401).json({
+      error:
+        "You must be signed in with a mozilla.com email address to access this page",
```

Comment:
Let's not give away that someone could do this with an @mozilla.com email address. Make an attacker have to crawl thru our source code to figure that out.